package.json

```diff
@@ -69,7 +69,7 @@
   },
   "x0": {
     "title": "Compositor x0",
-    "_basename": "/x0",
+    "basename": "/x0",
```

Comment:
This change allows `basename` to be configured in the `x0` field without affecting the dev server